python/tests/plugins/lsp_server/definition_constants.py

```diff
@@ -84,12 +93,23 @@
             when=[f"{TEST_SERVICE_THREE_NAME} receives a {TEST_SCHEMA_C.name} request"],
             then=[f"{TEST_SERVICE_THREE_NAME} returns the {TEST_SCHEMA_C.name} data in the response, untouched"],
         )
-    ]
+    ],
 )
 TEST_SERVICE_THREE = create_model_definition(TEST_SERVICE_THREE_NAME, behavior=[TEST_SERVICE_THREE_BEHAVIOR])
 
 TEST_DOCUMENT_NAME = "test.aac"
-TEST_DOCUMENT_CONTENT = f"{YAML_DOCUMENT_SEPARATOR}\n".join([TEST_SCHEMA_A.to_yaml(), TEST_SCHEMA_B.to_yaml(), TEST_SERVICE_ONE.to_yaml()])
+TEST_DOCUMENT_WITH_ENUM_NAME = "enum_test.aac"
+TEST_DOCUMENT_CONTENT = f"{YAML_DOCUMENT_SEPARATOR}\n".join(
+    [TEST_SCHEMA_A.to_yaml(), TEST_SCHEMA_B.to_yaml(), TEST_SERVICE_ONE.to_yaml()]
+)
```

Comment:
Looks like your editor may be set to a smaller line length than we have configured - please double-check it.

```suggestion
TEST_DOCUMENT_CONTENT = f"{YAML_DOCUMENT_SEPARATOR}\n".join([TEST_SCHEMA_A.to_yaml(), TEST_SCHEMA_B.to_yaml(), TEST_SERVICE_ONE.to_yaml()])
```